modules/modules.nix

```
@@ -112,6 +112,7 @@ let
     (loadModule ./services/getmail.nix { condition = hostPlatform.isLinux; })
     (loadModule ./services/gnome-keyring.nix { })
     (loadModule ./services/gpg-agent.nix { })
+    (loadModule ./services/grobi.nix { })
```

Since the module uses systemd it is Linux only:
```suggestion
    (loadModule ./services/grobi.nix { condition = hostPlatform.isLinux; })
```